docs/docs/faq.md

```diff
@@ -15,13 +15,14 @@ We know that JupyterLab is where many data scientist experiement and prove out t
 
 Yes, you can run NB Defense in your CI pipline using the [NB Defense CLI](./getting-started/cli.md)! Use the CLI in your CI pipelines to scan repositories, or multiple notebooks at a time. 
 
-### What is special about NB Defense when many security tools offer similar functionality?
+### What is unique about NB Defense when many security tools offer similar functionality?
 
-NB Defense is special because it allows you to scan Jupyter Notebooks. We provide both a [JupyterLab Extension](./getting-started/jupyter-lab-extension.md) that you can use to scan notebooks while you're working, and a [CLI](./getting-started/cli.md) that you can use to scan groups of notebooks or repositories. Using both of these tools, you can scan your notebooks for personally identifiable information (PII), secrets, common exposures and vulnerabilities (CVEs), and non permissive licenses. 
+
+NB Defense allows you to scan Jupyter Notebooks. Currently most security tools do not support Notebook `.ipynb` formatted files. NB Defense fills this gap. We provide both a [JupyterLab Extension](./getting-started/jupyter-lab-extension.md) that you can use to scan notebooks within Jupyterlab environment, and a [CLI](./getting-started/cli.md) that you can use to scan groups of notebooks or repositories. Using both of these tools, you can scan your notebooks for personally identifiable information (PII), secrets, common exposures and vulnerabilities (CVEs), and non permissive licenses.
```

Comment:
## Current
that you can use to scan notebooks within Jupyterlab environment

## Recommended
that you can use to scan notebooks within a JupyterLab environment

Reply:
Perhaps instead of saying they don't support .ipynb files, state that they aren't rendering findings in a way that a Jupyter user understands. Specifically they will report an issue on a line of a JSON file, not within a specific cell or output.